scripts/generator/graphql_generator/csharp/SDK/Android/AndroidNativeCheckout.cs.erb

```diff
@@ -60,16 +61,15 @@ namespace <%= namespace %>.SDK.Android {
             var requiresShipping = checkout.requiresShipping();
 
             #if !SHOPIFY_MONO_UNIT_TEST
-            using (var androidPayCheckoutSession = new AndroidJavaObject("com.shopify.unity.buy.AndroidPayCheckoutSession")) {
-                // TODO: Replace with mono object for receiving callbacks with.
-                object[] args = { "testing", merchantName, key, pricingLineItemsString, currencyCodeString, countryCodeString, requiresShipping };
-                if (androidPayCheckoutSession.Call<bool>("checkoutWithAndroidPay", args)) {
-                    // TODO: Do I need to do anything on the Unity side after we've created the session
-                } else {
-                    // TODO: Create more meaningful error?
-                    var error = new ShopifyError(ShopifyError.ErrorType.NativePaymentProcessingError, "Unable to create Android Pay session.");
-                    OnFailure(error);
-                }
+            AndroidPayCheckoutSession = new AndroidJavaObject("com.shopify.unity.buy.AndroidPayCheckoutSession");
```

Comment:
If `AndroidPayCheckoutSession` exists you don't want to create a new one every time. Any reason to why we may want to keep this resource alive, instead of using `using`?